test/functional/RasapiTest/test.xml

```diff
@@ -42,53 +40,61 @@
 			(Luckily the ant manual gives the java task you need to launch ant:
 			http://ant.apache.org/manual/running.html#viajava )
 		-->
-		
+
 		<java jvm="${test.java.home}/bin/java"
-			classname="org.apache.tools.ant.launch.Launcher"
-			fork="true"
-			failonerror="true"
-			dir="${basedir}"
-			timeout="900000"
-			taskname="test">
+				classname="org.apache.tools.ant.launch.Launcher"
+				fork="true"
+				failonerror="true"
+				dir="${basedir}"
+				timeout="900000"
+				taskname="test">
 			<classpath>
-				<pathelement location="${ant.home}/lib/ant-launcher.jar"/>
+				<pathelement location="${ant.home}/lib/ant-launcher.jar" />
 			</classpath>
-			<jvmarg value="-showversion"/>
-			<jvmarg value="-Xmx512M"/>
-			<arg value="-buildfile"/>
-			<arg file="${ant.file}"/>
-			<arg value="test"/>
+			<jvmarg value="-showversion" />
+			<jvmarg value="-Xgcpolicy:optthruput" />
```

Comment:
This one is the ant processing, it doesn't really need to change. Even the original value -Xm128m is fine here. On the other hand, it's not harmful to change it, this process doesn't (shouldn't) create any core files.

Reply:
Thanks for the reminder. I think this should be reverted to the original heap size (128M).